package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "username-sync",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
you can leave this `1.0.1` release tooling will increment on release, but thanks!